samples/linecounter/BlobProcessorHost.cs

```diff
@@ -60,9 +62,7 @@ public async Task StartAsync(CancellationToken cancellationToken)
 
         private async Task ProcessEvent(ProcessEventArgs eventArgs)
         {
-            // Note there is a bug with the HasEvent property in Preview 6, so check if Data is not null as work around.
-            // https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample08_EventProcessingHeartbeat.cs#L85
-            if (eventArgs.Data != null)
+            if (eventArgs.HasEvent)
             {
                 var totalCount = 0;
                 var cancellationToken = _cancellationTokenSource.Token;
```

Comment:
Apologies, I'm a bit late to the party.   Not critical, but you may want to consider upgrading line 70 to use the `eventArgs.Data.EventBody` member _(type `BinaryData`)_,  as that's the one that we'd like to highlight going forward.  `Body` is now a hidden member for back-compat only.

Reply:
Or, I can take a note to loop around and throw a PR out for it at some point this week.   Either way.  😄